src/main/content/blog.html

```diff
@@ -1,65 +1,132 @@
+            <h2 class="blog_subtitle">Never miss a post!</h2>
+            <p class="blog_subtext">Be sure to follow <a class="orange_link_light_background" href="https://twitter.com/OpenLibertyIO" target="_blank">@openlibertyio</a> on Twitter or subscribe to our <a class="orange_link_light_background" href="https://openliberty.io/feed.xml" target="_blank">RSS feed</a>.</p>
+            <img id="ufo_images_desktop" class="d-none d-lg-block" src="/img/blog_images_desktop.svg" alt="ufo and clouds">
+            <img id="ufo_images_tablet" class="d-none d-md-block d-lg-none" src="/img/blog_images_tablet.svg" alt="ufo and clouds">
         </div>
-        {% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
-        {% for post in site.posts %}
-        <div class="row blog_post_row">
-            <div class="col-xl-8 offset-xl-2 col-xs-12 blog_post_column">
-                <div class="blog_post_content">
-                    {% if post.redirect_link %}
-                        <h2 class="blog_post_title"><a href="{{ post.redirect_link }}" class="blog_post_title_link">{{ post.title | escape }}</a></h2>
-                    {% else %}
-                        <h2 class="blog_post_title"><a href="{{ post.url | relative_url }}" class="blog_post_title_link">{{ post.title | escape }}</a></h2>
-                    {% endif %}
-                    <div class="blog_post_author_data_container">
-                        <img src="{{ post.author_picture }}" class="blog_post_author_image" alt="Blog author picture">
-                        <a class="blog_post_author_name" href="{{ post.author_github }}">{{ post.author }}</a>
-                        <span class="blog_post_date">on {{ post.date | date: date_format }}</span>
-                    </div>
-                    <p class="blog_post_paragraph">
-                        {% assign continue_text = ' [Continued on ' %}
-                        {% if post.blog_description %}
-                            {{ post.blog_description }}
-                            {% if post.redirect_link %}
-                                <span id="continued_text">
-                                {% assign domain_name = post.redirect_link | remove: "https://" | remove: "http://" | remove: "www." %}
-                                {% assign clean_domain = domain_name | remove: '[' | remove: '"' %}
-                                {{ continue_text | append: clean_domain }}
-                                </span>
-                            {% endif %}
+        <div id="right_column" class="col-md-12 col-lg-8 offset-lg-4">
+
+            {% assign past_month_count = 0 %}
+            {% for post in site.posts %}
+
+                {% assign date_published = post.date | date:'%s' %}
+                {% assign date_now = 'now' | date:'%s' %}
+                {% assign date_difference = date_now | minus: date_published %}
+
+                {% if date_difference < 2592000 %}
```

Comment:
Could you please add a comment saying how 2592000 was derived e.g. 60*60*24*30 or at least say it's the number of seconds in a month in case someone looking at it later wonders what the number means.